carte_cli/main.py

```diff
@@ -58,5 +59,19 @@ def new_frontend(
     create_frontend_dir(name, init_admin=(not no_admin), sample_data=(not no_sample))
 
 
+@app.command("flatten")
+def flatten(
+    input_dir: str = typer.Argument(
+        ..., help="The source metadata directory, the same as the extraction output"
```

Comment:
what is the `...`?

Reply:
Special value in Python (haven't encountered it before this either). In Typer means the argument is required.